runner/src/redis-client/redis-client.ts

```diff
@@ -44,12 +44,15 @@ export default class RedisClient {
     await this.client.disconnect();
   }
 
-  async getNextStreamMessage (
+  async getStreamMessages (
     streamKey: string,
+    count = 1,
+    streamId = this.SMALLEST_STREAM_ID,
```

Comment:
```suggestion
    streamId = this.SMALLEST_STREAM_ID,
    count = 1,
```
Can we swap these, `count` seems less likely to change that `streamId`, so putting it after means we don't need to define `count` every time

Reply:
Ah good point. Switched!